docker-app/qfieldcloud/core/admin.py

```diff
@@ -1051,14 +1087,15 @@ class OrganizationAdmin(QFieldCloudModelAdmin):
         "email",
         "organization_owner",
         "date_joined",
+        "active_users",
     )
     list_display = (
         "username",
         "email",
         "organization_owner__link",
         "date_joined",
         # "storage_usage__field",
-        "active_users",
+        "active_users_count",
     )
```

Comment:
Did not yet add anything to changelist. Just renaming to use this variable name for the change view field.

Reply:
this is outdated I guess, so resolving.